eng/common/pipelines/templates/steps/verify-agent-os.yml

```diff
@@ -0,0 +1,38 @@
+# Template for all Python Scripts in this repository
+parameters: 
```

Comment:
@chidozieononiwu: The suggestion to remove this parameter was resolved.  Is this parameter supposed to still be here?

Reply:
We have the parameter still but we default it to `$(OSVmImage)` I think that approach works well and gives folks the flexibility to pass in another variable if needed when consuming the template.

Reply:
Got it, I didn't notice the default value.